__main__.py

```
@@ -0,0 +1,610 @@
+        elif args.mode == 'exploit':
+            run_standard_mode(args)
+        elif args.mode == 'hunt':
+            run_autonomous_mode(args)
+        else:
+            parser.print_help()
+            sys.exit(1)
+
+    except KeyboardInterrupt:
+        print("\n[!] Interrupted by user")
+        sys.exit(130)
+    except KeyboardInterrupt:
```

Ruff (0.12.2)

try-except block with duplicate exception `KeyboardInterrupt`

(B025)